### PR TITLE
mgr/dashboard: Fix notifications in user list and form

### PR DESCRIPTION
**Pull request by votdev**
The notification message title and text were inverted.

Signed-off-by: Volker Theile <vtheile@suse.com>

### CODE REVIEW

File: src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.ts [hunk 1]
```diff
@@ -99,8 +99,8 @@ export class UserListComponent implements OnInit {
     if (sessionUsername === username) {
       this.notificationService.show(
         NotificationType.error,
-        `You are currently authenticated with user "${username}".`,
-        'Cannot Delete User'
+        'Cannot Delete User',
```
Comment (Devp00l) [inline src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.ts]: Could you please use the wording of task messages here? (Also noted in `hacking.rst`) =>
`Failed to delete user ${username}`

File: src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.ts [hunk 2]
```diff
@@ -83,8 +83,8 @@ export class UserListComponent implements OnInit {
         this.modalRef.hide();
         this.notificationService.show(
           NotificationType.success,
-          `User "${username}" has been deleted.`,
-          'Delete User'
+          'Delete User',
```
Comment (Devp00l) [inline src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.ts]: Could you please use the wording of task messages here? (Also noted in `hacking.rst`) =>
`Deleted user ${username}`

File: src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.ts [hunk 3]
```diff
@@ -132,8 +132,8 @@ export class UserFormComponent implements OnInit {
       () => {
         this.notificationService.show(
           NotificationType.success,
-          `User "${userFormModel.username}" has been created.`,
-          'Create User'
+          'Create user "${userFormModel.username}"',
+          `User "${userFormModel.username}" has been created.`
```
Comment (Devp00l) [inline src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.ts]: This argument is now only used for error message descriptions. Please remove this line.

File: src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.ts [hunk 4]
```diff
@@ -209,8 +209,8 @@ export class UserFormComponent implements OnInit {
         } else {
           this.notificationService.show(
             NotificationType.success,
-            `User "${userFormModel.username}" has been updated.`,
-            'Edit User'
+            'Edit user "${userFormModel.username}"',
+            `User "${userFormModel.username}" has been updated.`
```
Comment (Devp00l) [inline src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.ts]: This argument is now only used for error message descriptions. Please remove this line.

File: src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.ts [hunk 5]
```diff
@@ -83,8 +83,8 @@ export class UserListComponent implements OnInit {
         this.modalRef.hide();
         this.notificationService.show(
           NotificationType.success,
-          `User "${username}" has been deleted.`,
-          'Delete User'
+          'Delete user "${username}"',
+          `User "${username}" has been deleted.`
```
Comment (Devp00l) [inline src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.ts]: This argument is now only used for error message descriptions. Please remove this line.

**Comment by votdev** [comment]
Jenkins retest this please

**Pull request merged**